src/FSharpSource.targets

```diff
@@ -181,6 +181,9 @@
     <DefineConstants>$(DefineConstants);TODO_REWORK_SERVER</DefineConstants>
     <DefineConstants>$(DefineConstants);NO_LOADER_OPTIMIZATION</DefineConstants>
     <DefineConstants>$(DefineConstants);SILVERLIGHT_COMPILER_FSHARP_CORE</DefineConstants>
+    <DefineConstants>$(DefineConstants);NETSTANDARD1_5</DefineConstants>
```

Comment:
We just need one of these defines since the code generated by FsSrGen for .NET Core uses ``#if NETSTANDARD1_5 || NETSTANDARD1_6 || ...``